### ISSUE TITLE
Minecraft Exit Code 255

### ISSUE THREAD

**Issue by waterdragon78** [cabaletta/baritone#2212]
## Some information
Operating system: MacOS (Beta:11.0.1) UWU OWO
Java version: 8
Minecraft version: 1.16.2
Baritone version: Latest
Forge mods (if used): N o (uwu)

## Exception, error or logs
[debug.log](https://github.com/cabaletta/baritone/files/5578875/debug.log)

Linux: `~/.minecraft/logs/`

Windows: `%appdata%/.minecraft/logs/`

Mac: `/Library/Application\ Support/minecraft/logs/`

## How to reproduce
Its with this one schematic and idk if its for any others, but just start building anything (i have all materials) and wait.

## Modified settings
default but allowinventory is true (Le  O W O)

## Final checklist OWO OWO OWO U W U
- [x] I know how to properly use check boxes 
- [x] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x ] I have included logs, exceptions and / or steps to reproduce the issue.
- [ ] I have not used any OwO's or UwU's in this issue. ( n e v e r )

**Comment by randomairborne** [comment]
Did you build with `./gradlew build -Pbaritone.forge_build`?

**Comment by randomairborne** [comment]
also the uwus and owos are a bit annoying.

**Comment by waterdragon78** [comment]
sowwy, but i didn't build I downloaded the Baritone-api-forge.jar and i tried impact client, however they both had the same issue.

**Comment by randomairborne** [comment]
99% its a Big Sur issue, then

**Comment by waterdragon78** [comment]
Nope, I’m afraid that this happened on Mojave too, and on windows (duel boot setup)

**Comment by waterdragon78** [comment]
Hewo?

**Comment by randomairborne** [comment]
are you loading a singleplayer world?
